google/cloud/storage/client.h

```diff
@@ -228,8 +228,12 @@ struct ClientImplDetails;
  *
  * The default policies are to continue retrying for up to 15 minutes, and to
  * use truncated (at 5 minutes) exponential backoff, doubling the maximum
- * backoff period between retries. Likewise, the idempotency policy is
- * configured to retry all operations.
+ * backoff period between retries.
+ *
+ * By default, all operations are treated as idempotent. The client library
+ * sends the `x-goog-gcs-idempotency-token` header with a unique value for
+ * each group of retried RPCs. The service will not execute a request with the
+ * same value for this header and send the previous response.
```

Comment:
That isn't as clear as it could be, methinks.  How about something like, "If the server receives a request with a token that it has previously seen, it will simply resend the matching response."

Reply:
Thanks, s/server/service/ but otherwise stole your text.